server/lib/tuist_web/controllers/api/bundles_controller.ex

```diff
@@ -20,6 +21,148 @@ defmodule TuistWeb.API.BundlesController do
 
   tags ["Bundles"]
 
+  operation :index,
+    summary: "List bundles for a project",
+    operation_id: "listBundles",
+    parameters: %{
+      account_handle: [
+        in: :path,
+        type: :string,
+        required: true,
+        description: "The handle of the account."
+      ],
+      project_handle: [
+        in: :path,
+        type: :string,
+        required: true,
+        description: "The handle of the project."
+      ],
+      git_branch: [
+        in: :query,
+        type: :string,
+        required: false,
+        description: "Filter bundles by git branch."
+      ],
+      page: [
+        in: :query,
+        type: :integer,
+        required: false,
+        description: "Page number for pagination."
+      ],
+      page_size: [
+        in: :query,
+        type: :integer,
+        required: false,
+        description: "Number of items per page."
+      ]
+    },
+    responses: %{
+      ok:
+        {"List of bundles", "application/json",
+         %Schema{
+           type: :object,
+           properties: %{
+             data: %Schema{
```

Comment:
hmm, this looks outdated as the `json` actually returns `bundles` key. I'm surprised the compilation would succeed, though, under these conditions 🤔 have you regenerated the OpenAPI yml?